analyzer/flow_client.go

```diff
@@ -110,7 +110,7 @@ func (c *FlowClientWebSocketConn) Close() error {
 // Connect to the WebSocket flow server
 func (c *FlowClientWebSocketConn) Connect() error {
 	authOptions := NewAnalyzerAuthenticationOpts()
-	authAddr := common.NormalizeAddrForURL(c.url.Hostname())
+	authAddr := common.NormalizeIPForUrl(c.url.Hostname())
```

Comment:
Hostname can return a domain name and even if we resolve the address earlier in the call flow I would prefer to keep addr to support domain name.

Reply:
done